File: source/SEP3 - test (Rider)/WebServices Unit Test/Model/Rate.cs
namespace Model
{
    public class Rate
    {
        public int GameId { get; set; }
        
        public string User { get; set; }
        
        public int Rating { get; set; }

        public override string ToString()
        {
            return GameId + " " + User + " " + Rating;
        }
    }
}

File: source/SEP3 - tier 2 (IntelliJ)/src/main/java/org/EvaAndTheSovietHouseholdAppliances/model/Rate.java
package org.EvaAndTheSovietHouseholdAppliances.model;

/**this class serves as a Data Object for Rating*/
public class Rate
{

    /**id of the game which this rating is associated with*/
    public int GameId;
    /**name of the user which this rating is associated with*/
    public String User;
    /**rating value*/
    public int Rating;

    /**initialises all instance variables*/
    public Rate(int gameId, String user, int rating)
    {
        GameId = gameId;
        User = user;
        Rating = rating;
    }

    public String toString()
    {
        return GameId + " " + User + " " + Rating;
    }
}
